autoload/vista/parser/ctags.vim

```diff
@@ -95,8 +95,10 @@ function! vista#parser#ctags#FromExtendedRaw(line, container) abort
   if a:line =~# '^!_TAG'
     return
   endif
-  " Prevent bugs when a:line is all whitespace
-  if a:line =~# '^\s*$'
+  " Prevent bugs when a:line is all whitespace or doesn't contain any tabs
+  " (can't be parsed).
+  if a:line =~# '^\s*$' || stridx(a:line, '\t') == -1
+    echom "Vista.vim: Error parsing ctags output: '" . a:line . "'"
```

Comment:
I'm not sure people woule love to this message.

Reply:
No-one likes to see error messages, but there's no log system or debug option in place.
The alternative is Vista doesn't work, and no-one knows why.

If you suggest a debug option name, I'll wrap it in a check

Reply:
If this cases happens only when you are using Exuberant ctags, I have mentioned that it's unsupported in the doc.

I'm happy to accept a debug option if someone implements a more complete debug system, but not for this only specical case.